pages/_app.js

```diff
@@ -6,16 +6,20 @@ function MyApp({ Component, pageProps }) {
   // Custom hooks setting user user details - here in the _app.js so it can be accessed acrosss all componenets!
   const [learnerName, setLearnerName] = useLocalStorageState(
     "learner-name",
-    null
+    ""
   ) || [null, null];
-  const [font, setFont] = useLocalStorageState("font", null) || [null, null];
-  const [size, setSize] = useLocalStorageState("font-size", null) || [
+  const [font, setFont] = useLocalStorageState("font", "Comic") || [null, null];
+  const [size, setSize] = useLocalStorageState("font-size", "medium") || [
     null,
     null,
   ];
   const [background, setBackground] = useLocalStorageState(
     "background",
-    null
+    "#fff"

```

Comment:
Can we change the default background colour to ``#F8F5F4`` instead? It's just off-white so a little less harsh on the eyes. (I got that colour from the [BDA website](https://www.bdadyslexia.org.uk/advice/employers/creating-a-dyslexia-friendly-workplace/dyslexia-friendly-style-guide) :)